slide-deck.js

```diff
@@ -589,68 +593,119 @@ class slideDeck extends HTMLElement {
 
   move = (by) => {
     const to = (this.#getActive() || 0) + by;
-    this.goTo(to);
+    this.toSlide(to);
   };
 
-  goToSaved = () => {
-    this.goTo(this.#slideFromStore());
+  next = () => this.move(1);
+  previous = () => this.move(-1);
+
+  toSavedSlide = () => {
+    this.toSlide(this.#slideFromStore());
   }
 
+  // --------------------------------------------------------------------------
+  // keyboard control
+
   // Detect Ctrl / Cmd modifiers in a platform-agnostic way
   #cmdOrCtrl = (event) => event.ctrlKey || event.metaKey;
 
-  #keyEventActions = (event) => {
-    // exit from blank slide
-    if (this.hasAttribute('blank-slide')) {
+  #escToBlur = (event) => {
+    if (event.key === 'Escape') {
       event.preventDefault();
-      this.removeAttribute('blank-slide');
-      return;
+      event.target.blur();
+      return true;
+    }
+  }
+
+  #isPrivateKeydown = (event) => {
```

Comment:
`type` may not be set- I don't think we need to add an explicit fallback to `text` here because it's not in `clickOnly`.
```suggestion
        const type = event.target.getAttribute('type')?.toLowerCase();
```